app/models/jellyfish_demo/service/compute.rb

```diff
@@ -0,0 +1,7 @@
+class Service < ActiveRecord::Base
```

Comment:
This is missing the `JellyfishDemo` module namespace.

I think all of your service classes are.